### DISCUSSION TITLE
[Combobox] How to make the first item to always be the active one?

### DISCUSSION THREAD

**Discussion by martinszeltins** [tailwindlabs/headlessui#1242] [Help]
I am building a Command Palette using Headless UI Combobox component but I can't figure out how to make the first item to always be the active/selected one. This way I can simply hit "Enter" after typing and it would select the first option.

**Comment by tobihagemann** [comment]
Duplicate of #1178

**Reply by mohammadzolfali5** [reply]
> - # []()~~****~~well come

**Comment by RobinMalfait** [comment]
Hey! Thank you for your suggestion!
Much appreciated! 🙏

This pr should fix that: https://github.com/tailwindlabs/headlessui/pull/1279
